Piviot (#2)
fullsail/capstone-Demetrius-B: docs/readme.md

## 1.2. Problem

**Primary Problem**

There isn't a way for the primary audience to exposing their inventory to local vapors.

**Secondary Problem**

The lack of information redarding specific vape juices and inventory visability.

## 1.3. Solution

VapeLocate is a competitor, and they address the problem of finding a vape shop.

An inventory management system that allows business owners to import their existing inventory, receive statistics about products and manage their account.

## 1.4. Unique Value Proposition

Business owners will be able to expose their inventory to local vapers.
Local Vapers will be able to find local stores that carry a specific vapor juice that's in stock.

## 1.5. User Stories
* As a Business owner I can input my inventory into the platform.

> * As a Business owner I would not like to not have to configure in detail each item being imported (Picture, Flavor Profile, Categories, marketing blurb, etc.).

Reviewer: ```suggestion
* As a Business owner, I would like to not have to configure in detail each item being imported (Picture, Flavor Profile, Categories, marketing blurb, etc.). So that the importing process is hand free, once configured.
```

> * As a Business owner I only want my Vape Juice included into the platform.

Reviewer: Does not make sense.

```suggestion
 
```

* As a Business owner I would like to use an existing inventory system to management inventory.
* As a local vapor I would like to know what flavors are in-stock at stores near me.

> * As a local vapor when 'follow/watch' become available, I would like to be notified.

Reviewer: ```suggestion
* As a local vapor when followed / watched flavors become available, I would like to be notified.
```

* As a local vapor I would like to find local stores that carry a vapor juice based upon specific criteria (brand, flavor, etc.).
* As a Business owner I would like to be notified if a significant number of local vapors are interested in one of my vape flavors.
* As a Business I can manage my agit ccount via a web portal

### 1.5.1. Functional Requirements (Features)

> > * As a Business owner I can input my inventory into the platform.

Reviewer: ```suggestion

```

> >   * System must allow the user to import their existing inventory.

Reviewer: ```suggestion

```

> > * As a Business owner I would not like to not have to configure in detail each item being imported (Picture, Flavor Profile, Categories, marketing blurb, etc.).

Reviewer: ```suggestion
> * As a Business owner, I would like to not have to configure in detail each item being imported (Picture, Flavor Profile, Categories, marketing blurb, etc.). So that the importing process is hands-free, once configured.
```

> >   * System must configure each item being imported.

Reviewer: ```suggestion
>   * The System must import data from an external source.
>   * The System must have an interface for the business owner to identify how the importing data should convert into the System's data structure, creating a data map.
>   * System must apply the data map to any new data imports for that business owner.

```

> > * As a Business owner I only want my Vape Juice included into the platform.

Reviewer: This runs counter to your local vapor's needs.
```suggestion

```
